rst_files/getting_started.rst

```diff
@@ -206,16 +176,36 @@ If it's a long file, however, you have the alternative of
 The present working directory can be found by executing the command ``pwd()``
 
 
+Using QuantEcon Lecture Packages
+-------------------------------------------
+
+To use the curated set of packages in the QuantEcon lecture notes, 
+copy the following text into a notebook cell, and hit ``Shift-Enter`` to run the cell
+
+   .. literalinclude:: /_static/includes/deps.jl    
```

Comment:
Here is where we already have run the dependencies

Reply:
As I said, I am very confused right now with your suggested changes.  Can you restart your PR by putting in the many excellent edits which don't have to do with flow changes in the desktop setup vs. jupyter setup steps?

Then we can see what you are proposing for the reorganization of the other material in isolation.  I may agree with a lot of it now that I have hopefully made the point that the setup and usage sections need to be distinct.